build.sbt

```diff
@@ -25,6 +31,22 @@ def jsonDependencies(scalaVersion: String) = Seq(
   logback % Test
 ) ++ jacksons ++ specsBuild.map(_ % Test)
 
+// Common settings 
+import com.typesafe.sbt.SbtScalariform._
+import scalariform.formatter.preferences._
+
+val previousVersion = "2.6.0-M1" // first from this separate repo
```

Comment:
@cchantep The point is so the code exists to enable MiMa, and when we create a new branch for the release we can activate it to test binary compatibility.